diff -Naurp php-5.5.9/ext/xmlrpc/xmlrpc-epi-php.c php-5.5.9.oden/ext/xmlrpc/xmlrpc-epi-php.c
--- php-5.5.9/ext/xmlrpc/xmlrpc-epi-php.c	2014-02-05 11:00:36.000000000 +0100
+++ php-5.5.9.oden/ext/xmlrpc/xmlrpc-epi-php.c	2014-02-12 11:13:04.873805413 +0100
@@ -71,7 +71,7 @@
 #include "ext/date/php_date.h"
 #include "php_ini.h"
 #include "php_xmlrpc.h"
-#include "xmlrpc.h"
+#include <xmlrpc-epi/xmlrpc.h>
 
 static int le_xmlrpc_server;
 
