diff -Naurp php-5.5.9/ext/bz2/bz2_filter.c php-5.5.9.oden/ext/bz2/bz2_filter.c
--- php-5.5.9/ext/bz2/bz2_filter.c	2014-02-05 11:00:36.000000000 +0100
+++ php-5.5.9.oden/ext/bz2/bz2_filter.c	2014-02-12 11:29:43.416861248 +0100
@@ -276,6 +276,8 @@ static php_stream_filter_status_t php_bz
 				data->strm.avail_out = data->outbuf_len;
 				data->strm.next_out = data->outbuf;
 				exit_status = PSFS_PASS_ON;
+			} else if (status == BZ_OK) {
+				break;
 			}
 		}
 	}
