diff -Naurp php-5.5.9/configure.in php-5.5.9.oden/configure.in
--- php-5.5.9/configure.in	2014-02-05 11:00:36.000000000 +0100
+++ php-5.5.9.oden/configure.in	2014-02-12 11:30:53.599865172 +0100
@@ -295,9 +295,6 @@ if test "$GCC" = "yes"; then
     GCC_MAJOR_VERSION=`$CC --version | $SED -n '1s/[[^0-9]]*//;1s/\..*//;1p'`
     ;;
   esac
-  if test $GCC_MAJOR_VERSION -ge 4; then
-    CFLAGS="$CFLAGS -fvisibility=hidden"
-  fi
 fi
 
 case $host_alias in
