diff -Naurp php-5.5.9/php.ini-development php-5.5.9.oden/php.ini-development
--- php-5.5.9/php.ini-development	2014-02-12 11:31:47.296868175 +0100
+++ php-5.5.9.oden/php.ini-development	2014-02-12 11:31:59.264868844 +0100
@@ -1362,6 +1362,11 @@ session.use_cookies = 1
 ; http://php.net/session.use-only-cookies
 session.use_only_cookies = 1
 
+; This option forces new session ID when browser supplied uninitialized session
+; ID. By enabling this option, module prevents session fixation based on adoption.
+; http://php.net/session.use-strict-mode
+session.use_strict_mode = 0
+
 ; Name of the session (used as cookie name).
 ; http://php.net/session.name
 session.name = PHPSESSID
diff -Naurp php-5.5.9/php.ini-production php-5.5.9.oden/php.ini-production
--- php-5.5.9/php.ini-production	2014-02-12 11:31:47.296868175 +0100
+++ php-5.5.9.oden/php.ini-production	2014-02-12 11:31:59.264868844 +0100
@@ -1362,6 +1362,11 @@ session.use_cookies = 1
 ; http://php.net/session.use-only-cookies
 session.use_only_cookies = 1
 
+; This option forces new session ID when browser supplied uninitialized session
+; ID. By enabling this option, module prevents session fixation based on adoption.
+; http://php.net/session.use-strict-mode
+session.use_strict_mode = 0
+
 ; Name of the session (used as cookie name).
 ; http://php.net/session.name
 session.name = PHPSESSID
